libdnf5-cli/output/repo_info.cpp

```diff
@@ -22,10 +22,25 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include "key_value_table.hpp"
 #include "utils/string.hpp"
 
+#include "libdnf5-cli/tty.hpp"
 #include "libdnf5-cli/utils/units.hpp"
 
+#include <json.h>
```

Comment:
This is a nitpick but I would prefer to include as little as possible, only: `json-c/json_object.h` should suffice but feel free to leave it as it is.

Reply:
Yep, agree.